External/Plugins/ASCompletion/Completion/ASComplete.cs

```diff
@@ -3777,6 +3817,94 @@ static public ASResult GetExpressionType(ScintillaControl sci, int position, boo
             }
         }
 
+        static private MemberList GetTypeParameters(MemberModel model)
+        {
+            var retVal = new MemberList();
```

Comment:
It was mainly done that way for readability, but sure, no problem.

I also found a problem in other part of code. PR in a few minutes.